docs/data-cookbooks/browse-generation.md

```diff
@@ -42,17 +42,15 @@ You should have data available for Cumulus to ingest in an S3 bucket that matche
 
 For the purposes of this entry, we will be using a pre-configured  MOD09GQ version 006 CMR collection.    If you'd prefer to utilize the example processing code, using mocked up data files matching the file naming convention will suffice, so long as you also have a matching collection setup in CMR.
 
-If you'd prefer to ingest another data type, you will need to generate a processing lambda (see [Build Processing Lambda](#build-processing-lamvda) below).
+If you'd prefer to ingest another data type, you will need to generate a processing lambda (see [Build Processing Lambda](#build-processing-lambda) below).
 
 -----------
 
 ## Configure Cumulus
 
 ### CMR
 
-Visit the [CMR configuration documentation](../deployment/config_descriptions#cmr) for instructions on CMR integration and configuration.
-
-These configuration keys will be used in the CmrStep/PostToCmr Lambda function below.
+(Need section on updating CMR configuration in TF)
```

Comment:
As the config_descriptions page is going away, we need to find another relevant reference

Reply:
Also, I'm not sure what the reference would be here, since part of our idea was to only document variable definitions in the child module

Reply:
Yeah, indeed, that's part of what I wanted to discuss.      Probably a reference to the new config document/keys.  🤔

Reply:
65592de3b